catalog/UMT/UMT_Climate_IrrMapper_RF_v1_2.jsonnet

```diff
@@ -0,0 +1,105 @@
+local id = 'UMT/Climate/IrrMapper_RF/v1_1';
+local subdir = 'UMT';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'IrrMapper Irrigated Lands',
+  version: '1.1',
```

Comment:
please update the version to 1.2 and make any other changes to reflect what's new in this version